test/conversion.test.js

```diff
@@ -51,6 +97,17 @@ describe('Curl converter should', function() {
 		done();
 	});
 
+	it('throw an error if POST body option is given with HEAD and PUT and without --get/-G', function(done) {
+		convert({
+			type: 'string',
+			data: 'curl -I -T "./example.txt" http://example.com -d "a=b"'
```

Comment:
-T with --data doesn't throw an error i.e. `curl -T "./Desktop/test.txt" -d 'sdsd' http://httpbin.org/put -v` won't throw an error

Reply:
@shreys7 Yeah!! 
I've changed the logic and test now.